index.html

```diff
@@ -0,0 +1,170 @@
+            </li>
+          </ul>
+          <div class="hamburger">
+            <img src="./images/hamburger-menu.png" alt="hambuger" />
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <h3 class="main_sub_title">"Homes For the Masses"</h3>
+      <h1 class="main_title">AFFORDTABLE HOUSING EXHIBITION 2022</h1>
+      <p class="main_intro">
+        A sustainable housing delivery system with emphasis on easy access to
+        home ownership for the Nigeria Poeple in an environment where basic
+        infrastructural amenities are readily avaliable.
+      </p>
+      <p class="date">2022.01.16(MON) ~ 18(WED)</p>
+      <p class="main_venue">&copy; Liberty Conference Hall, Auja-Nigeria</p>
+    </main>
+    <section class="activity">
+      <h3 class="activity_title">Main Activities</h3>
+      <div class="bar home_bar"></div>
+      <ul class="activity_menu">
+        <li class="activity_menu_item">
+          <img src="./images/forum.png" alt="Lecture" /><span
+            class="program_title"
+            >Introduction</span
+          >
+          <div class="brief">
+            A brief summary of the all stages for the exhibition and to take
+            full advantage of the opportunity it presents.
+          </div>
+        </li>
+        <li class="activity_menu_item">
+          <img src="./images/exhibition.png" alt="Exhibition" /><span
+            class="program_title"
+            >Exhibition</span
+          >
+          <div class="brief">
+            Presenting different categories of low cost housing to the
+            participant
+          </div>
+        </li>
+        <li class="activity_menu_item">
+          <img src="./images/forum.png" alt="Forum" /><span
+            class="program_title"
+            >Discussion</span
+          >
+          <div class="brief">
+            Open discussion between management and participant on area that need
+            more clearity
+          </div>
+        </li>
+        <li class="activity_menu_item">
+          <img
+            class="workshop"
+            src="./images/exhibition.png"
+            alt="Workshop"
+          /><span class="program_title">Workshop</span>
+          <div class="brief">
+            Lecture and discussion with expert from the financial section for
+            enlightenment on different housing scheme
+          </div>
+        </li>
+        <li class="activity_menu_item">
+          <img
+            class="networking"
+            src="./images/forum.png"
+            alt="Networking"
+          /><span class="program_title">Networking</span>
+          <div class="brief">
+            Opportunity to meeting new business partner for participant that
+            willing to invest.
+          </div>
+        </li>
+      </ul>
+      <p class="connect">Join Exquisite Residence Exhibition 2022</p>
+    </section>
+    <div class="guest_speaker"></div>

```

Comment:
- [ ] Kindly apply the changes requested by previous reviewer like the font-size, padding, margin and proportion of images against the description section, etc.
####Implementation:
![speakers-section-img-distorted](https://user-images.githubusercontent.com/44057863/169396212-a54a6460-38fc-4d6a-a7bd-cf064ff66aa0.png)